### PR TITLE
Route "straightening"

### PR DESCRIPTION
**Pull request by Bryukh**

### CODE REVIEW

File: verification/src/referee.py [hunk 1]
```diff
@@ -316,6 +321,7 @@ def clear_from_map(self, item):
         size = item.size * self.GRID_SCALE
         fill_square(self.map_grid, item.coordinates[0] * self.GRID_SCALE - size // 2,
                     item.coordinates[1] * self.GRID_SCALE - size // 2, size, 1)
+        self.map_graph = grid_to_graph(self.map_grid)
```
Comment (oduvan) [inline verification/src/referee.py]: why not self.create_route_graph() ?

Comment (Bryukh) [inline verification/src/referee.py]: one line function?

Comment (oduvan) [inline verification/src/referee.py]: https://github.com/CheckiO/checkio-empire-battle/pull/45/files#diff-bca170bcf6047f32b2c0ef53fc9b328eR314

Comment (Bryukh) [inline verification/src/referee.py]: hm, intersting and why I dont remeber this.

**Comment by oduvan** [comment]
ok, merge

**Pull request merged**